frontend/src/main/components/Courses/CourseTable.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/CoursesUtils"
+import { hasRole } from "main/utils/currentUser";
+
+export default function CourseTable({ courses, currentUser }) {
+
+    // Stryker disable all : hard to test for query caching
+    const deleteMutation = useBackendMutation(
+        cellToAxiosParamsDelete,
+        { onSuccess: onDeleteSuccess },
+        []
+    );
+    // Stryker enable all
+    
+    // Stryker disable next-line all : TODO try to make a good test for this
+    const deleteCallback = async (cell) => { deleteMutation.mutate(cell); }
+
+    const columns = [
+        {
+            Header: 'id',
+            accessor: 'id', 
+        },
+        {
+            Header: 'Enrollment Code',
+            accessor: 'enrollCd',
+        },
+        {
+            Header: 'Personal Schedule ID',
+            accessor: 'psId',
+        },
+    ];
+
+    const columnsIfUser = [
+        ...columns,
+        // ButtonColumn("Edit", "primary", editCallback, "PersonalSchedulesTable"),
```

Comment:
Best to remove this until is ready to be implemented.   I'll overlook it if this is the only thing on the code review that needs to be changed.